### DISCUSSION TITLE
client pings, but there's no entry for page: /findroom

### DISCUSSION THREAD

**Discussion by DuffyWang926** [vercel/next.js#14411] [Help]
# Bug report

## Describe the bug
when type in the cmd nom run dev。
sometime, it will show the error' client pings, but there's no entry for page:'
in the chrome: the moduleId of webpack_require is undefined;
sometime, the error in cmd doesn't  show,but the error in the chrome is exit.
and wait for 1-3 minute, the page is normal.

<img width="584" alt="image" src="https://user-images.githubusercontent.com/31502024/85226309-76db2480-b409-11ea-984c-382f83714075.png">

git address 
https://github.com/DuffyWang926/sharingHomesSSR.git
## System information

- OS: catalina
- Browser (if applies) chrome
- Version of Next.js: 9.1.4
- Version of Node.js: 12.18.1

**Comment by timneutkens** [comment]
I'd start by upgrading to the latest version of Next.js
